keps/sig-instrumentation/3498-extending-stability/README.md

```diff
@@ -311,19 +341,18 @@ This should not affect upgrade/rollback paths.
 
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
 
-`Alpha` metrics will be recategorized as `Internal`.
+No.
 
 ### Monitoring Requirements
 
 ###### How can an operator determine if the feature is in use by workloads?
 
-You can determine this by seeing if workloads depend on any Kubernetes control-plane metrics. If they do, they are using this feature.
+Dependence on any Kubernetes control-plane metrics implies that they are using this feature.
```

Comment:
L366 - I hope it's not a missing metric, right?

Reply:
I mean if the stability framework is not functioning properly, then that is one of many metrics that will not appear.

Reply:
But that's not what the question is about.

The question is: "are there any metrics that we didn't implement and in the ideal world we should".

From what I can tell, there aren't any such metrics - the `registered_metrics_total` metric actually exists, right?

Reply:
Ah that's true.